src/elements/content-explorer/ItemGridThumbnail.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import getProp from 'lodash/get';
+import { getIcon } from '../common/item/iconCellRenderer';
+import './ItemGridThumbnail.scss';
+
+type Props = {
+    item: BoxItem,
+    token: Token,
+};
+
+const ItemGridThumbnail = ({ item, token }: Props) => {
+    let url;
+    const entries = getProp(item, 'representations.entries');
+    if (getProp(entries, '[0].status.state') === 'success') {
+        const entry = getProp(entries, '[0]');
+        const rep = getProp(entry, 'representation');
```

Comment:
In general, we like to avoid abbreviations. Increased readability is the priority, it all gets minified by webpack anyway